test/lib/common_test.js

```diff
@@ -485,6 +505,42 @@ exports.itCreatedKubernetesPipelineFilesWithExpectedContent = function (opts) {
   })
 }
 
+//
+// VSI deployment
+//
+exports.itCreatedVSIFilesWithExpectedContent = function (opts) {
+  opts = opts || {}
+  var applicationName = opts.applicationName || 'appname'
+
+  it('created VSI files', function () {
+    assert.file(exports.vsiFiles)
+  })
+
+  it('created toolchain files', function () {
+    assert.file(exports.bluemixFiles)
+  })
+
+  it('debian control file contains expected content', function () {
+    assert.fileContent(exports.debianControl, `Package: ${applicationName}-0.0`)
+  })
+
+  it('debian install file contains expected content', function () {
+    assert.fileContent(exports.debianInstall, `usr/src/${applicationName}`)
+  })
+
+  it('terraform start file contains expected content', function () {
+    assert.fileContent(exports.terraformStart, `./${applicationName}`)
+  })
+
+  it('terraform variables file contains expected content', function () {
+    assert.fileContent(exports.terraformVariables, `default = "${applicationName}-01"`)
+  })
+
+  it('terraform variables file contains expected content', function () {
```

Comment:
oops, accidental copy-paste error... Although not sure if we should keep this test for Jenkinsfile (see other review comment)